src/ecdsa/keys.py

```diff
@@ -902,14 +904,30 @@ def from_der(cls, string, hashfunc=sha1):
               publicKey  [1] BIT STRING OPTIONAL
             }
 
+        `publicKey` field is ignored completely (errors, if any, in it will
+        be undetected).
+
         The only format supported for the `parameters` field is the named
         curve method. Explicit encoding of curve parameters is not supported.
+        In the legacy sslay format, this implementation requires the optional
+        `parameters` field to get the curve name. In PKCS #8 format, the curve
+        is part of the PrivateKeyAlgorithmIdentifier.
+
+        The PKCS #8 format includes this object as the `privateKey` field
```

Comment:
using `this` for subject of a sentence two paragraphs before feels a bit vague to me